src/java.desktop/macosx/classes/sun/lwawt/LWComponentPeer.java

```diff
@@ -984,7 +988,25 @@ public boolean requestFocus(Component lightweightChild, boolean temporary,
 
     @Override
     public final Image createImage(final int width, final int height) {
-        return getLWGC().createAcceleratedImage(getTarget(), width, height);
```

Comment:
Where is this method called from? If it is invoked by the RepaintManager, it might be better to create the appropriate back buffer there - or alternatively, you can implement OffScreenImage similar to VolatileImage. Then it will cover all platforms.

Also note that the RepaintManager uses different round logic.